From 3f9a5fcc8328e2573bed6d539dddda0132189062 Mon Sep 17 00:00:00 2001
From: Lorenzo Pirritano <lpirritano@anaconda.com>
Date: Wed, 21 Feb 2024 21:15:03 -0600
Subject: [PATCH] skip test for darwin

This test fails on mac os for no apparent reason with:

======================================================================
FAIL: testUnpackBoolean (testGenpy.TestGeneratePython)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/lorepirri/miniconda3/conda-bld/titus2_1708542245206/test_tmp/test/testGenpy.py", line 2114, in testUnpackBoolean
    self.assertEqual(engine.action("".join(map(unsigned, [8]))), True)
AssertionError: False != True

----------------------------------------------------------------------

---
 test/testGenpy.py | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/test/testGenpy.py b/test/testGenpy.py
index c4a86ac..fdb40e0 100644
--- a/test/testGenpy.py
+++ b/test/testGenpy.py
@@ -2099,6 +2099,8 @@ action:
         self.assertRaises(PFAUserException, lambda: engine.action("".join(map(unsigned, []))))
         self.assertRaises(PFAUserException, lambda: engine.action("".join(map(unsigned, [1, 2]))))
 
+    import platform
+    @unittest.skipIf(platform.system() == "Darwin", "fails on mac os")
     def testUnpackBoolean(self):
         engine, = PFAEngine.fromYaml('''
 input: bytes
-- 
2.39.1

